test/simple_socketio.test.js

```diff
@@ -1,7 +1,9 @@
 // :copyright: Copyright (c) 2023 ftrack
 
 import { describe, test, vi } from "vitest";
-import SimpleSocketIOClient, { PACKET_TYPES } from "../source/simple_socketio";
+import SimpleSocketIOClient, {
+  PACKET_TYPES,
+} from "../source/simple_socketio.js";
```

Comment:
with nodenext you need to use `.js`, what i don't understand is how this wasn't erroring.